uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -188,6 +189,30 @@ private function createDefaultRelationshipTypes() {
     }
   }
 
+  /**
+   * Making All Currencies Available for new installations
+   */
+  private function makeAllCurrenciesAvailable() {
+    $dao = CRM_Core_DAO::executeQuery('SELECT * from civicrm_currency');
+    while ($dao->fetch()) {
+      $currencyExists = civicrm_api3('OptionValue', 'get', [
+        'value' => $dao->name,
+      ]);
```

Comment:
You need to add the option_group_id to the parameter to fetch the option value. In Civi it is possible for an option value with same names to exist and they will belong to different option groups. So for example we can have `GBP` belonging as an option value to another option group.